hooks/useNavigation.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
these really dynamic records... i think they could match to the schema of our translations? I didn't attempt.

Reply:
I'm not sure, but this file will be removed in the future, so that's fine.

We might want to open a separate issue just for the typings of this file? But I'm quite sure we might get rid of the navigation.json file